### PR TITLE
server admin option (for 4.2.11) required corresponding PRC change

### PR DESCRIPTION
**Pull request by d-w-moore**
4.2.11 was failing on the ticket tests which had passed for 4.2.10 prior to the addition of admin -M into server ticket routines.

Thus, this introduces  adaptive class definition for the TicketAdminRequeset so that Tickets work for both 4.2.11 and pre-4.2.11 servers.

### CODE REVIEW

**Comment by trel** [comment]
@michael-conway this seems important for jargon and @iychoi for the go-irodsclient too

**Comment by d-w-moore** [comment]
Tested, will squash and we can merge, provided this looks good on review....

**Comment by d-w-moore** [comment]
Regarding ADMIN_KW, ... I have tested the ability to use it from PRC. It works - if we'd like this reflected in tests and docs and such, just let me know @korydraughn  @trel

**Comment by korydraughn** [comment]
Adding a test(s) is the best thing to do. If all ticket operations go through the same path in the PRC, all we need is a test that proves the ADMIN_KW works.

**Comment by d-w-moore** [comment]
ok, think I have a test in mind

File: irods/test/ticket_test.py [hunk 1]
```diff
@@ -71,6 +73,27 @@ def tearDown(self):
...
+
+        with helpers.make_session() as ses:
+            alices_tickets = [t[TicketQuery.Ticket.string] for t in ses.query(TicketQuery.Ticket).filter(TicketQuery.Owner.name == 'alice')]
+            self.assertEqual(len(alices_tickets),N_TICKETS)
+            for s in alices_tickets:
+                Ticket( ses, s ).delete(**{kw.ADMIN_KW:''})
+            alices_tickets = [t[TicketQuery.Ticket.string] for t in ses.query(TicketQuery.Ticket).filter(TicketQuery.Owner.name == 'alice')]
+            self.assertEqual(len(alices_tickets),0)
+
+
```
Comment (korydraughn) [inline irods/test/ticket_test.py:96]: is this test demonstrating deleting another user's (alice) tickets using a different iRODS account (i.e. rods)?

does `helpers.make_session()` represent a rodsadmin user?
if so, let's add some comments stating that. it's not clear who is deleting the tickets.

**Comment by trel** [comment]
looks good - please squash the desired amount, add issue number(s)

File: irods/test/ticket_test.py (hunk 1 above)
Comment (d-w-moore) [reply irods/test/ticket_test.py]: @korydraughn  Yes, did add the comments, but it probably needs to be documented more generally how these tests are to be run.  Whether in CI or standalone, irods.test.helpers.make_session( ) tends to be used to create the default "from environment" session with an account iinit'ed for rods.

**Comment by d-w-moore** [comment]
Done!

**Pull request merged**